includes/course-theme/class-sensei-course-theme-option.php

```diff
@@ -122,15 +124,19 @@ public function should_use_sensei_theme() {
 
 		$course_id = \Sensei_Utils::get_current_course();
 
-		if ( null === $course_id ) {
+		if ( empty( $course_id ) ) {
 			return false;
 		}
 
-		if ( ! $this->has_sensei_theme_enabled( $course_id ) ) {
-			return false;
+		if ( self::has_sensei_theme_enabled( $course_id ) ) {
+			return true;
+		}
+
+		if ( Sensei_Course_Theme::is_sensei_theme_preview_mode_allowed( $course_id ) ) {
```

Comment:
I saw that it was a suggestion to rename this method, but while reading it in this context, it doesn't make much sense. I think the previous method name would work better. cc @LavaGolem WDYT?

Reply:
So which name do you folks prefer? 😄 Let me know when you decide and I'll change it right away.

Reply:
Actually, since it's already under the class `Sensei_Course_Theme`, `is_preview_mode` would probably be enough. WDYT?

Reply:
Yep. Updated. 👍  cfe65c8b10eb0d4e8c229590e9c399b8313ca0ea